recipes/libtool/libtool_2.4.2.oe

```diff
@@ -1,19 +1,12 @@
-LICENSE = "GPL-2.0+"
-LICENSE_${PN}-libltdl = "LGPL-2.1+"
+require ${PN}-${PV}.inc
 
-require libtool.inc
+inherit autotools-autoreconf
 
-addtask bootstrap before configure after fetch unpack patch stage
-do_bootstrap[dirs] = "${S}"
+RECIPE_TYPES = "machine cross sdk-cross"
 
-do_bootstrap () {
-	if [ "${RECIPE_TYPE}" != "native" ]; then
-		./bootstrap
-	fi
-}
+DEPENDS += "native:help2man"
```

Comment:
I guess this is correct, but...

OE-lite/core recipes must not depend on anything from OE-lite/base.  It must be possible to build OE-lite/core by itself.  In this case, the solution could be to convince libtool to not build man-pages (by default), and thus avoid this dependency.